valkey/templates/metrics-svc.yaml

```diff
@@ -21,6 +21,9 @@ spec:
       port: {{ .Values.metrics.service.ports.http }}
       protocol: TCP
       targetPort: metrics
+      {{- if and .Values.metrics.service.appProtocol (semverCompare ">=1.20.0-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
I think it is safe to assume that nobody is (or should) be running a 2020 version of Kubernetes. No need to check for capabilities.

Reply:
Ok. Wasn't clear to me which version of Kubernetes were supported by this chart so I played it safe. 😄